tasks/checkInsolvencyAtMaturity.ts

```diff
@@ -41,7 +37,14 @@ task(
   );
 
   let noneInsolvent = true;
-  for (const marginEngineAddress of marginEngineAddresses) {
+  for (const poolName in poolAddresses) {
```

Comment:
(Why) is this an improvement? Seems like the previous approach had an advantage of being network agnostic and also ignoring margin engines with no relevant positions?

Reply:
I agree, keeping it network agnostic is a good idea.